<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>698</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>698</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Отчет о ЧС</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog{
background-color:rgb(255, 255, 127);
}

QPushButton{
background-color:#55aaff;
border-style:outset;
padding:2px;
border-width:5px;
border-radius:7px;
border-color:#55aaff;
}
</string>
  </property>
  <widget class="QTableWidget" name="tWReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>681</width>
     <height>311</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
  </widget>
  <widget class="QComboBox" name="ComboBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>по убыванию</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>390</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pBtnReport">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/report.png</normaloff>../../../Downloads/report.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBtnClose">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/close.png</normaloff>../../../Downloads/close.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2021</number>
   </property>
   <property name="maximum">
    <number>2030</number>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>80</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>21</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
